diff --git a/node_modules/@rainbow-me/rainbowkit/.DS_Store b/node_modules/@rainbow-me/rainbowkit/.DS_Store
new file mode 100644
index 0000000..6384e57
Binary files /dev/null and b/node_modules/@rainbow-me/rainbowkit/.DS_Store differ
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/.DS_Store b/node_modules/@rainbow-me/rainbowkit/dist/.DS_Store
new file mode 100644
index 0000000..53a5c3f
Binary files /dev/null and b/node_modules/@rainbow-me/rainbowkit/dist/.DS_Store differ
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/chunk-ZESJAA3L.js b/node_modules/@rainbow-me/rainbowkit/dist/chunk-ZESJAA3L.js
index a89044a..c785314 100644
--- a/node_modules/@rainbow-me/rainbowkit/dist/chunk-ZESJAA3L.js
+++ b/node_modules/@rainbow-me/rainbowkit/dist/chunk-ZESJAA3L.js
@@ -3453,12 +3453,12 @@ function ConnectButton({
       transition: "default"
     }, /* @__PURE__ */ React42.createElement(Box, {
       as: "button",
-      background: "accentColor",
-      borderRadius: "connectButton",
+      backgroundImage: "linear-gradient(to right, #3898FF, #7A70FF)",
+      borderRadius: "6",
       boxShadow: "connectButton",
       color: "accentColorForeground",
-      fontFamily: "body",
-      fontWeight: "bold",
+      fontFamily: "EarthOrbiter",
+      fontWeight: "normal",
       onClick: openConnectModal,
       paddingX: "14",
       paddingY: "10",
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/components/.DS_Store b/node_modules/@rainbow-me/rainbowkit/dist/components/.DS_Store
new file mode 100644
index 0000000..881057f
Binary files /dev/null and b/node_modules/@rainbow-me/rainbowkit/dist/components/.DS_Store differ
